release-notes.md

```diff
@@ -8,6 +8,13 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 Please take a moment to fill out a [short survey](https://www.surveymonkey.com/r/AEPDocs) on how we can better assist you with enabling Adobe Experience Cloud solutions and services on your mobile apps.
 
+## January 25, 2020
+
+The following updates were made in this release:
+
+**Android Analytics 1.2.3 and iOS Analytics 2.2.2**
+
+* `requestEventIdentifier` is now appended to all  non-track events so that Lifecycle (or other extension  events sent to Analytics) can be viewed with rich detail in Project Griffon.
```

Comment:
- Looks like there might be an extra space between "extension" and events"
- Please add "that are" between "events" and "sent to"
- In addition to the main RN page, can you please also add the final text here: https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/adobe-analytics-release-notes

Reply:
@rekrishn I can make these changes. However, I don't understand why we are duplicating release notes in two separate areas.